generator/src/test/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGeneratorTest.java

```diff
@@ -110,4 +99,28 @@ public void testGeneratesValidExpressions() throws Exception {
     }
   }
 
+  private List<Stmt> makeStmtList(BasicType basicType,
+                                  ShadingLanguageVersion shadingLanguageVersion) {
```

Comment:
^ Can you give this a more descriptive name, to reflect the fact that it makes a list of assignments of mutated expressions to fresh variables?  While we don't want ridiculously long names, I'm OK with long-ish names that will continue to make sense when a test class like this one gets larger.

Reply:
OK, what about `makeVariableListAndAssignMutatedExpression`?